examples/docker-bookinfo.yaml

```diff
@@ -30,6 +30,8 @@ gateway:
     - controller
     - registry
   container_name: gateway
+  command: /usr/bin/run_filebeat.sh
+
```

Comment:
Why does the gateway have to run the run_filebeat.sh? Isn't it the stock sidecar?